downloadfile.py

```diff
@@ -0,0 +1,10 @@
+import os.path
+import requests
+url='https://samples.adsbexchange.com/readsb-hist/2022/05/01/235745Z.json.gz'
```

Comment:
don't need this.

Reply:
Do you want me to remove these? When I remove them I am not able to run the function.